src/index.ts

```diff
@@ -15,7 +15,7 @@ export {
 export {FieldAnalyser} from './fieldAnalyser/fieldAnalyser';
 export {DocumentBuilder} from './documentBuilder';
 export * from './securityIdentityBuilder';
-export {PlatformEnvironment, Region, PlatformUrlOptions} from './environment';
+export {PlatformEnvironment, Region, PlatformOptions} from './environment';
```

Comment:
Should this be considered a breaking change?

Reply:
Hmmmmm I hadn't thought about that... However, I'm not sure if there's actually a use-case for an implementer 
 importing `PlatformOptions` since it's only used in the constructors, which all have default values. I'm personally not _super_ worried, but it is something that could happen 🤔